syllabus/examples/training_scripts/cleanrl_procgen_plr.py

```diff
@@ -392,6 +406,12 @@ def get_value(obs):
                     writer.add_scalar("charts/episodic_length", item["episode"]["l"], global_step)
                     if curriculum is not None:
                         curriculum.log_metrics(writer, global_step)
+
+                        # track return for individual tasks
+                        idx = info.index(item)
+                        multiprocessing_sync_wrapper_envs = envs.venv.venv.envs # extract envs of class MultiProcessingSyncWrapper from envs of class VecNormalize, which has access to task id
+                        episode_task = multiprocessing_sync_wrapper_envs[idx]._latest_task
+                        curriculum.update_on_episode(item["episode"]["r"], item["episode"]["l"], episode_task, args.env_id)
```

Comment:
This isn't necessary, the curriculum sync wrapper will call this automatically with the correct data.